### PR TITLE
Initializing borrowing app

### PR DESCRIPTION
**Pull request by Nyxan**
Знімок екрана 2024-06-14 142034
Знімок екрана 2024-06-14 142118

Знімок екрана 2024-06-14 140904
Знімок екрана 2024-06-14 141106

### CODE REVIEW

**Review by panicua — CHANGES_REQUESTED** [review]
NVIDIA_Share_yK3TTGB0TN

You need to implement all the stuff:
GET list method should have:
GET:              borrowings/?user_id=...&is_active=...  - get borrowings by user id and whether is borrowing still active or not.

Borrowing should have this POST method:
POST: 	          borrowings/<id>/return/ 		- set actual return date (inventory should be made += 1)

Also, every time person borrows a book, the book invetory should change to -1 of what it was, so if book inventory were 15, when person takes 1 book it it should be 14

**Comment by Nyxan** [comment]
I oriented myself for tasks in documentation
Screenshot_2024-06-14-15-03-54-829-edit_com android chrome

**Review by panicua — APPROVED** [review]
In this case seems like well done. (4th paragraph from task)

File: borrowing_app/views.py [hunk 1]
```diff
@@ -1,3 +1,18 @@
-from django.shortcuts import render
+from rest_framework import viewsets
+
+from borrowing_app.models import Borrowing
+from borrowing_app.serializers import BorrowingSerializer, BorrowingListSerializer, BorrowingDetailSerializer
+
+
+class BorrowingViewSet(viewsets.ModelViewSet):
+    queryset = Borrowing.objects.all()
```
Comment (panicua) [inline borrowing_app/views.py:8]: won't you need a ```def get_querry_set()``` method in this case? borrowing model has many-to-many with books and many-to-one with user, you'll probably need to use select_related, prefetch_related so it doesn't create duplicated querys

**Review by nicksetrakov — APPROVED** [review]

**Pull request merged**